alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -44,9 +65,21 @@ public function postDelete(PropelPDO $con = null)
 	{
 		$this->addTrackEntryInfo(TrackEntry::TRACK_ENTRY_EVENT_TYPE_DELETE_MEDIA_SERVER, __METHOD__);
 		
-		$entrySaved = $this->updateLiveEntryData(null);
-		if (!$entrySaved)
-			$this->indexLiveEntry();
+		$liveEntry = $this->getLiveEntry();
+		if($liveEntry)
+		{
+			if($this->getServerType() === EntryServerNodeType::LIVE_PRIMARY)
+			{
+				$liveEntry->setPrimaryServerNodeId(null);
+				
+				$entryServerNodes = EntryServerNodePeer::retrieveByEntryId($this->getEntryId());
+				if($liveEntry->getCurrentBroadcastStartTime())
+					$liveEntry->setCurrentBroadcastStartTime(0);
+			}
+			
+			if(!$liveEntry->save())
+				$liveEntry->indexToSearchIndex();
```

Comment:
Save should generally work - I think that it is better to call $liveEntry->indexToSearchIndex(); within the if ($this->getServerType() === EntryServerNodeType::LIVE_PRIMARY) condition